test/unit/module/algo/algorithm/transform_keep_if_and_remove_generic_test.hpp

```diff
@@ -24,7 +24,7 @@ template <typename T, typename Algo, typename Op>
 void transform_keep_if_generic_test_aligned_ptr(eve::as<T>, Algo alg, Op op)
 {
   using e_t = eve::element_type_t<T>;
-  alignas(64) std::array<e_t, 23> data;
```

Comment:
Because later we have eve::as_aligned(data.begin()) without cardinal, so it uses default cardinal, that is for short type is (128/16)*8=64. (the `*8` appears as at maximum, we use 8 real vector registers), that will result in alignment 64 * 2 = 128 byte.

This could be also fixed if I set  `eve::as_aligned(data.begin(), eve::fixed<8>{})` (or whatever value to be sure that it will not be greater that 64)